src/routes/matchHistoryRoutes.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express'
+
+function getSummoner(){
+    const router = express.Router()
+    router.get('/summoner', getSummonerHandler)
```

Comment:
Now that you have the version of the api that accepts the summoner name, just get rid of the /summoner api and the getSummonerHandler